src/modules/ekf2/ekf2_main.cpp

```diff
@@ -286,6 +290,7 @@ class Ekf2 final : public ModuleBase<Ekf2>, public ModuleParams
 	uORB::Publication<wind_estimate_s>			_wind_pub{ORB_ID(wind_estimate)};
 	uORB::PublicationData<vehicle_global_position_s>	_vehicle_global_position_pub{ORB_ID(vehicle_global_position)};
 	uORB::PublicationData<vehicle_local_position_s>		_vehicle_local_position_pub{ORB_ID(vehicle_local_position)};
+	uORB::PublicationData<vehicle_odometry_s>		_vehicle_aligned_visual_odometry_pub{ORB_ID(vehicle_aligned_visual_odometry)};
```

Comment:
No need for a new message.

Reply:
@TSC21 I think this would be cleaner than using instances to publish this, especially because you could potentially have multiple odometry sources. Also, we generally don't change a topic and re-publish it under a new instance, right?